.actor/input_schema.json

```diff
@@ -0,0 +1,31 @@
+{
+	"title": "Omio scraper",
+	"type": "object",
+	"schemaVersion": 1,
+	"properties": {
+		"from": {
+			"title": "From",
+			"description": "From location.",
+			"editor": "textfield",
+			"type": "string",
+			"prefill": "Brno"
+		},
+		"to": {
+			"title": "To",
+			"description": "To location.",
+			"editor": "textfield",
+			"type": "string",
+			"prefill": "Prague"
+		},
+		"date": {
+			"title": "Date",
+			"type": "string",
+			"description": "Select an absolute travel date in format YYYY-MM-DD. The date must be either today or in the future (past dates are not supported).",
+			"editor": "datepicker",
+			"dateType": "absoluteOrRelative",
+			"pattern": "^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[12]\\d|3[01])$|^(\\d+)\\s*(day|week|month|year)s?$",
+			"prefill": "27.3.2025"
```

Comment:
the value is invalid :slightly_smiling_face: 
![image](https://github.com/user-attachments/assets/83d3c11f-831f-449d-a0a4-7990d55dfe7a)